core/example/benchmark/CMakeLists.txt

```diff
@@ -1,7 +1,8 @@
+
 if(Cabana_ENABLE_Serial)
   add_executable(NeighborListMDPerfTest md_neighbor_perf_test.cpp)
   target_link_libraries(NeighborListMDPerfTest cabanacore)
 endif()
 
-add_executable(PeakFlops Cabana_peakflops.cpp)
-target_link_libraries(PeakFlops cabanacore)
+#add_executable(PeakFlops Cabana_peakflops.cpp)
```

Comment:
If this is in the new performance test directory let's just get rid of these lines.